src/EventAction.cc

```diff
@@ -284,35 +287,80 @@ void EventAction::ReOrderTrackIds(Int_t subId) {
     }
 }
 
-void EventAction::SetTrackSubeventIDs() {
+int EventAction::SetTrackSubeventIDs() {
     Int_t nTracks = restG4Event->GetNumberOfTracks();
+    Double_t timeDelay = restG4Metadata->GetSubEventTimeDelay();  // in unit us
 
-    Double_t timeDelay = restG4Metadata->GetSubEventTimeDelay() * REST_Units::s;
-
-    vector<Double_t> fTrackTimestampList;
-    fTrackTimestampList.clear();
-
+    // reorder tracks
+    std::map<int, TRestG4Track*> tracks; 
```

Comment:
How it is this compiling? Should it be `TRestGeant4Track` ... there is a kind of dictionary?

Reply:
I forgot to rename it when copying the code from v2.2

Reply:
Ok, but then I dont understand how commit e76e69f5 undergoes compilation and passes the pipeline at build stage.

https://gitlab.cern.ch/rest-for-physics/restg4/-/jobs/13684941